module/VuFind/src/VuFind/Controller/AbstractBase.php

```diff
@@ -712,6 +712,14 @@ protected function setFollowupUrlToReferer(bool $allowCurrentUrl = true, array $
         if ($mrhuNorm === $refererNorm) {
             return;
         }
+        // if  is the stored current url of the lightbox
```

Comment:
I'm not sure I understand this comment, and `if  is` suggests to me that maybe there are extra or missing words.